packages/cfd/src/Containers/cfd-financial-stp-real-account-signup.tsx

```diff
@@ -61,7 +60,10 @@ const CFDFinancialStpRealAccountSignup = (props: TCFDFinancialStpRealAccountSign
     const state_index = step;
     const height = 'auto';
     let is_mounted = React.useRef(true).current;
-    const { need_poi_for_vanuatu } = getIdentityStatusInfo(props.account_status);
+    const { need_poi_for_vanuatu, need_poi_for_bvi_labuan_maltainvest } = getAuthenticationStatusInfo(
+        props.account_status
```

Comment:
```suggestion
        props.account_status
```
Can you de-structure this at the top? Makes it clearer what props are involved
```
const { account_status, refreshNotifications, authentication_status, fetchStatesList } = props;
```